components/contribution-flow/index.js

```diff
@@ -7,7 +7,6 @@ import { themeGet } from '@styled-system/theme-get';
 import { debounce, findIndex, get, isNil, omit, pick } from 'lodash';
 import { defineMessages, FormattedMessage, injectIntl } from 'react-intl';
 import styled from 'styled-components';
-import { URLSearchParams } from 'universal-url';
```

Comment:
I didn't know we had this, I've been using the `objectToQueryString` helper. I'll migrate the code using it.